src/queue/PlaylistEdit.cxx

```diff
@@ -98,6 +98,14 @@ playlist::AppendSong(PlayerControl &pc, DetachedSong &&song)
 		throw PlaylistError(PlaylistResult::TOO_LARGE,
 				    "Playlist is too large");
 
+#if 0
```

Comment:
All this commit does is add disabled code. Bogus commit.

Reply:
Just for caryaudio.